### DISCUSSION TITLE
[Tokenized Index Fund] Strategy and Introduction of Candidate Framework : Balancer Managed Pools and Cow AMM

### DISCUSSION THREAD

**Discussion by hackfisher** [ktondao/KtonDAO#38] [General]
### Strategic Focus Due to Resource and Workforce limitations

Due to limited resources and workforce, we must clearly and explicitly define the following strategies:

1. **Focus on Frontend UI Development**  
   R&D efforts will be confined to improving the frontend UI to enhance accessibility and strengthen the brand, with an emphasis on simplicity.  
   *(This means we need a **Frontend Contributor**. Refer https://github.com/ringecosystem/collaboration/issues/99 , RingDAO ecosystem is OK to sponsor a grant with no more than 2.5K USD, UI design and requirement is still in draft and TBD after market&solution research)*

2. **No New Smart Contract or Protocol Development**  
   We will not develop new smart contracts or protocols due to the associated security risks and auditing costs. Instead, we will leverage existing, mature protocols such as **Balancer Managed Pools**.  
   For now, we will seek smart contract, security, and deployment support from @hujw77 ,  who is under RingDAO Ecosystem Technical Support WG.

3. **Coordinator Role**  
   A Coordinator will be required to facilitate communication and governance, market research and market strategy, knows the business and protocols and act as a coordinator between stakeholders.  
   **@SasoLithops** has been invited to take on this role.

---

### Resources for Understanding Balancer Managed Pools

For an introduction to **Balancer Managed Pools**, please refer to [Balancer Managed Pools Documentation](https://docs.balancer.fi/concepts/pools/managed.html). The core of this business lies in **risk control**. Understanding the characteristics of the pools, their parameters, and management strategies is critical to grasping the product's framework.  
Please ensure that you thoroughly review and study these materials before making any recommendations.

---

### Insights on Cow AMM

While **[Cow AMM](https://balancer.fi/pools/cow)** performs exceptionally well in mitigating **Loss-Versus-Rebalancing (LVR)**, it currently supports only two fixed pool ratios (50/50 and 80/20).  
This means it is currently limited to fixed-weighted index compositions and has not yet evolved to support Managed Pools. In this area, avoid pushing forward prematurely and patiently wait for the maturity of the underlying infrastructure and external Solver networks.

**Comment by hackfisher** [comment]
For the first pool and target assets, instead of relying on high-yield DeFi assets that require a high trust threshold, why not focus on actively traded and widely popular assets, such as meme tokens? For instance, a pool featuring the _**Top 5 Meme Tokens on Base**_ could be a good idea.

The Top 5 Meme Tokens on Base pool will consist of the five meme coins with the highest market capitalization or trading volume on the Base chain. Additionally, the weights can be periodically adjusted based on changes in market capitalization.

**Reply by sonoferin2000** [reply]
I personally don’t interact with memecoins, but I do see the logic in including them as part of the strategy. Their popularity and high trading activity make them a compelling choice for attracting attention and driving engagement. I have no opposition to this approach.
Buckle up, Captin Saso,, we are preparing for lift-off

**Reply by SasoLithops** [reply]
I also don't interact with meme coins. However I think if we go straight to meme index we will not be taken seriously. We will be looked similar like short term meme project. I do also agree with you both that this would be a popular move. So what do you think if we launch two indexes. A more stable one and meme one. The stable one could be for more Conservative (would have BTC, ETH, SOL, AVAX, DOT, KTON...) and Meme one ( Memes+some BTC and KTON(20% max)) All numbers and tokens up for debate.
What do you think?
Captain @sonoferin2000  to the moon;-)

**Reply by 2xJDubs** [reply]
Agree with Saso here that the Index should first focus on large cap long term projects.  Later on it can add some popular meme tokens as a small part of the holding.

**Comment by SasoLithops** [comment]
I am also posting on which index I helped before:

https://index.ion.wtf/

Here we discussed before putting it up for governance:

https://forum.osmosis.zone/u/ibcindex/summary

Please check it out.
I think webpage could be similar. I will go more in depth of all that next week.

**Reply by sonoferin2000** [reply]
I’m cool with adding a memecoin index—it makes sense with all the interest around them. Personally, I’d be more into an index for Polkadot parachains with some DOT in it, or Kusama parachains with some KSM. Our tokens would fit right in and feel like a good match there. I don’t think KTON needs to be in every index, but I’ll leave that up to you all to decide.

**Reply by sonoferin2000** [reply]
we still have pink? perhaps we could include that in the mix.i don't know, i don't pay any attention to those tokens, ill let more knowledgable people make that call.

**Reply by SasoLithops** [reply]
All up to us all. You might be right in not including Kton in every index. Might seem too maxi. I just felt we could bring some small BTC and KTON balance next to all memes in index. I take this from more risky savings funds which also have some bonds next to risky stocks...I could be wrong with traditional banking thinking:-) 
As for Polkadot and Kusama index, at this time they both seem kind of lost. I wouldn't know which projects to add, but I don't follow them that much as I used to.... I wouldn't do only them but might be interested in the future if Polkadot eco recovers. However we need to think how to include our sister CKTON for sure.

**Reply by SasoLithops** [reply]
I think WUD seems more popular than PINK. We do have some PINK in Ring DAO treasury left if I remember correctly. Someone who follows POlkadot and Kusama eco would be more then welcome to comment on this.

**Reply by sonoferin2000** [reply]
> All up to us all. You might be right in not including Kton in every index. Might seem too maxi. I just felt we could bring some small BTC and KTON balance next to all memes in index. I take this from more risky savings funds which also have some bonds next to risky stocks...I could be wrong with traditional banking thinking:-) As for Polkadot and Kusama index, at this time they both seem kind of lost. I wouldn't know which projects to add, but I don't follow them that much as I used to.... I wouldn't do only them but might be interested in the future if Polkadot eco recovers. However we need to think how to include our sister CKTON for sure.

@hackfisher, I have a question about CKTON. Are all CKTON tokens on the Crab network, or are there additional tokens on another chain? I know there was some rebranding with CRING and Crab. According to https://crab-scan.darwinia.network/token/0x0000000000000000000000000000000000000402 , CKTON shows a max supply of 222,982.233. However, if you add up all the holders listed there, it only totals 79,427.8453.

This discrepancy has me confused. Is the total supply figure a typo, or are those other tokens located on a different chain? I didn’t follow Crab closely in its early days, so I’m unsure if CKTON might have been included in a liquidity pool on a centralized exchange or something similar.

If those tokens exist elsewhere, maybe we could remind everyone that they can be migrated. I'd like to account for all CKTON since my assumptions have been based on the 79,427.8453 number. A discrepancy of 143,554.3877 CKTON is significant. If they’re circulating on another chain, could you please share the contract address so I can investigate further? or are the unretrivable and essentially considered burned?

**Reply by hackfisher** [reply]
CKTON is also substrate assets, working as ERC20 by Precompile, It is probably some inconsistent issues from blockscout explorer causing the holder datas inaccurate as it only index data from EVM events (missing if tokens were transferred via substrate extrinsic).

**Reply by sonoferin2000** [reply]
thanks for your quick response. I just want to ensure that the total supply figure I’m working with (79,427.8453 CKTON) is accurate for future considerations, and that no hidden tokens are waiting to be unlocked or migrated. is it safe to assume that?

**Comment by SasoLithops** [comment]
Also check this:

https://youtu.be/_hrqsOzyv3Q?si=kJmpBUlSqT9TF2he


Injective is EVM chain. So could be SOL memes also next to Base ones in index if we can connect to them.

**Comment by ahjxcrz** [comment]
I reviewed Balancer, and it's incredibly versatile, supporting multiple assets, making it a great tool for building index funds. Within the Polkadot ecosystem, we can leverage transaction volume and TVL data from Hydration to identify the most popular assets or trending memes.

Currently, based on TVL (excluding stablecoin pools), the leading assets are DOT, vDOT, WETH, ASTR, GLMR, CFG, and BNC. If we look at transaction volume, the top contenders are DOT, ASTR, and vDOT.

On the meme side, we could consider MYTH, WUD, and perhaps LOVA—though I’m sure you can guess why LOVA made the list! 😊

**Reply by SasoLithops** [reply]
Good points @ahjxcrz 

This is exactly where the problem lies. Too few interesting tokens od Polkadot. Maybe we could use Darwinia tech and supply Index with tokens from different ecosystems. Imagine having BTC, ETH, SOL, SUI, ASTR, ADA and so on all in one Index? I think this would be ground breaking as indexes if existing are only focused on one ecosystem.

**Comment by hujw77** [comment]
Hi @SasoLithops, I'm getting ready to test the managed pool on Base Network. Once I finish the testing, I'll put together a user-friendly manual that you can use to manage the pool later.

**Reply by SasoLithops** [reply]
Hi @hujw77!

Great news Echo. Thanky you so much for your effort. Looking forward to the manual:-)

**Comment by johneggleston** [comment]
Hey Everyone, as someone who has been looking into index projects myself. I actually love this idea. 

If you guys are indexing tokens by creating exotic LP pools with RING and KTON, here is something cool you can explore.  

Pairing to rugged projects to revive them and initiate CTOs - This can get especially interesting if the rugged project still has organic trading volume, as the supply of the rugged project can become rapidly constructed through the fees accrued through the LP mining.

**Reply by SasoLithops** [reply]
Hey John!

Glad to see you here. Initial idea is to keep it as simple as possible. Not creating exotic LP pools. I do hope however we will have many more different and also more risky indexes in the future.
